docs/src/modules/java/partials/topic-eventing.adoc

```diff
@@ -27,19 +27,32 @@ The Protobuf rpc method receiving these JSON messages must be set up to receive
 
```

Comment:
> Content-Type` attribute stating `application/json`

is that correct?

Reply:
I think so, from looking at the cloud event to any logic in the proxy.

Reply:
yes, that's' how I understand it. 

If I'm consuming some topic, the topic needs to have json so Akka Serverless recognises it as json, wrap it in an `Any` to deliver to my method.